packages/orbit-components/src/Box/consts.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+import theme from "../defaultTheme";
+
```

Comment:
Most of the defined constants are being used only in the stories, but a fragment of it is being imported into `normalize.js` helper function. Therefore it will increase the bundle size with unused stuff.

Would split it – leave `WIDTH_AND_HEIGHT` and `TOKENS` here, the rest can be moved to the stories file for playground purposes.